src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,28 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Options for the constructor
+     *
+     * @var array
+     */
+    protected $_options = [];
+
+    /**
      * Constructor.
      *
+     * Available option for $options is 'partialNullsPass' flag. 
+     * Set to true to allow passing of composite foreign keys where one or multiple keys are null.
```

Comment:
'Set to true to accept composite foreign keys where one or more nullable columns are null.'